### PR TITLE
DNM: Fix unstability while trying to update epel repositories

### LINKED ISSUE #2282: Fix CentOS random failures
I have seen this random failure
```
14:02:56 Adding repo from: https://download.docker.com/linux/centos/docker-ce.repo
14:02:58 CentOS-8 - Base                                 3.1 kB/s | 3.1 kB     00:01    
14:02:59 CentOS-8 - AppStream                            3.1 kB/s | 3.1 kB     00:01    
14:03:02 CentOS-8 - Extras                               693  B/s | 1.5 kB     00:02    
14:03:04 CentOS-8 - PowerTools                           1.9 kB/s | 4.3 kB     00:02    
14:03:06 Docker CE Stable - x86_64                       1.9 kB/s | 3.5 kB     00:01    
14:03:07 Extra Packages for Enterprise Linux Modular 8 -  14 kB/s |  18 kB     00:01    
14:03:09 Extra Packages for Enterprise Linux 8 - x86_64  9.7 kB/s |  19 kB     00:01    
14:03:13 Extra Packages for Enterprise Linux 8 - x86_64  1.3 MB/s | 5.5 MB     00:04    
14:03:14 Metadata cache created.
14:04:42 Error: Failed to synchronize cache for repo 'epel-modular'
14:04:42 Last metadata expiration check: 0:01:29 ago on Tue 04 Feb 2020 08:03:13 PM UTC.
14:04:42 No match for argument: docker-ce-
14:04:42 Error: Unable to find a match
14:04:43 Build step 'Execute shell' marked build as failure
```
It seems CentOS has some issues in trying to synchronize

### PR DESCRIPTION
**Pull request by GabyCT**
We have seen random failures while trying to update CentOS repositories.
This PR fixes that.

Fixes #2282

Signed-off-by: Gabriela Cervantes <gabriela.cervantes.tellez@intel.com>

### CODE REVIEW

File: .ci/setup_env_centos.sh [hunk 1]
```diff
@@ -18,6 +18,8 @@ else
   centos_version=$(grep VERSION_ID /usr/lib/os-release | cut -d '"' -f2)
 fi
 
+sudo -E yum -y clean all
```
Comment (grahamwhaley) [inline .ci/setup_env_centos.sh]: Do we know why it fixes it? Given afaik all the centos CIs will be running in clean deploys, it seems odd.

Comment (GabyCT) [reply .ci/setup_env_centos.sh]: not really, I am trying to implement what I see that has been suggested as a solution on the CentOS forums

**Pull request closed without merging**